### ISSUE TITLE
Broken AltGr keystrokes?

### ISSUE THREAD

**Issue by sprat** [fusion809/terminal-fusion#4]
### Description

It can't enter characters such as `~` or `@` in the terminal. In fact, all characters obtained by a `AltGr` keystroke are broken.

Note that I have a french keyboard, so I installed the `keyboard-localization` plugin and configured it with a `fr_FR` layout. With that setup, I can perfectly enter those characters in the editor views.
### Steps to Reproduce

Type `AltGr + 0`: nothing appear. A `@` character should appear.
### Versions

```
$ atom -version
Atom    : 1.10.2
Electron: 1.4.1
Chrome  : 53.0.2785.113
Node    : 6.5.0

$ apm -version
apm  1.12.7
npm  3.10.8
node 6.7.0
python 2.7.12
git 2.10.0

$ uname -r
4.7.5-1-ARCH
```

**Comment by fusion809** [comment]
For me entering `~` is by pressing <kbd>Shift</kbd>+<kbd>`</kbd> and it runs fine in my <code>terminal-fusion</code> terminal. Probably means this bug is isolated to your keyboard. Not sure how to fix it though. You wouldn't happen to have some idea as to how to fix it would ya?

**Comment by fusion809** [comment]
Plus my keyboard has no <kbd>AltGr</kbd> button, closest thing I have is <kbd>Alt</kbd>.

**Comment by sprat** [comment]
I've tested in a similar setup in my home machine running Linux Mint and Cinnamon desktop and it works. Will try to investigate tomorrow. But it's node 5.12.0 here.

**Comment by sprat** [comment]
In fact, I suspect the problem appear only with node v6. I've encountered the same problem with another plugin too, and many plugins are simply broken with node v6.

I've just installed the same version of Atom on my Arch Linux machine but with node v5.10 : the problem is gone and all the broken plugins I had found seem to work now.

It's a shame that Arch Linux ships Atom with node v6 which causes problems with many atom plugins.

As I encountered too many other problems to stay with an atom with node v6, I don't need a fix anymore ;)

It's now up to you to investigate the problem if you want ;)
